documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,27 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
-[repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
+Puppet Server is a required application that runs on the Java Virtual Machine (JVM). It performs the role of the master node that controls configuration information for one or more managed agent nodes
```

Comment:
1. Missing a period at the end here.
2. Since we're working on moving away from the "master" terminology, I wonder if there is a better way to describe this role.